src/Examine.Lucene/ValueTypeFactoryCollection.cs

```diff
@@ -71,7 +73,46 @@ public IFieldValueTypeFactory GetRequiredFactory(string valueTypeName)
+                                                                                         defaultAnalyzer,
+                                                                                         false,
+                                                                                         true,
+                                                                                         (readerReference, suggestionOptions, searchText) => FullTextType.ExecuteFuzzySuggester(readerReference, suggestionOptions, searchText, name, defaultAnalyzer))},
+                {FieldDefinitionTypes.FullTextSortableFuzzySuggestable, name => new FullTextType(name,
+                                                                                                 loggerFactory,
+                                                                                                 defaultAnalyzer,
+                                                                                                 true,
+                                                                                                 true,
+                                                                                                 (readerReference, suggestionOptions, searchText) => FullTextType.ExecuteFuzzySuggester(readerReference, suggestionOptions, searchText, name, defaultAnalyzer))},
+                {FieldDefinitionTypes.FullTextSpelling, name => new FullTextType(name,
+                                                                                 loggerFactory,
+                                                                                 defaultAnalyzer,
+                                                                                 false,
+                                                                                 true,
+                                                                                 (readerReference, suggestionOptions, searchText) => FullTextType.ExecuteDirectSpellChecker(readerReference, suggestionOptions, searchText, name, defaultAnalyzer)
+                                                                                 )},
+                {FieldDefinitionTypes.FullTextSortableSpelling, name => new FullTextType(name,
+                                                                                         loggerFactory,
+                                                                                         defaultAnalyzer,
+                                                                                         true,
+                                                                                         true,
+                                                                                         (readerReference, suggestionOptions, searchText) => FullTextType.ExecuteDirectSpellChecker(readerReference, suggestionOptions, searchText, name, defaultAnalyzer)
+                                                                                 )},
```

Comment:
This might be easier to view if the indentation/wrapping was the same as the rest of the configuration. (Not having every argument on a new line)